extensions/GlobalMessages/tests/GlobalMessages0.data.php

```diff
@@ -0,0 +1,970 @@
+'content-warning-body' => 'The wiki at $1 may contain objectionable content such as depictions of violence, sexual suggestion, dark humor parody, or other materials not intended for a general audience.
+
+In general, Fandom does not review nor do we endorse the content of this or any wiki. For more information about our policies, please visit [http://www.wikia.com/Terms_of_Use Wikia, Inc\'s Terms of Use].',
+// 37 --------------------------------------------------------------------------
+'cookie-policy-description' => 'In the EU, there is a law that users be notified of a web site\'s cookie usage upon arriving there for the first time. This extension allows Fandom to be in compliance with this law.',
+// 38 --------------------------------------------------------------------------
+'cookie-policy-notification-message' => 'Fandom uses cookies to give you the best experience on our websites. By continuing to use Fandom, we assume that you accept our use of cookies. [[homepage:Privacy_Policy#Cookies| Read more about cookies]]',
+// 39 --------------------------------------------------------------------------
+'create-account-new' => 'Create your Fandom Account',
+// 40 --------------------------------------------------------------------------
+'createpage-ve-body' => 'The article <b>$1</b> does not yet exist on this community. You can help out by adding a few sentences.',
+// 41 --------------------------------------------------------------------------
+'edit-account-closed-flag' => '<div class="errorbox" style="padding: 1em;">This account has been disabled globally by [[Help:Closing an account|user choice]], or by Fandom.</div>',
+// 42 --------------------------------------------------------------------------
+'editaccount-log-header' => 'This page lists changes made to user information by Fandom Staff.',
+// 43 --------------------------------------------------------------------------
+'editor-preference-help' => 'On this wiki, the default editor is always the classic editor. [[Help:Editing|Learn more]].',
+// 44 --------------------------------------------------------------------------
+'emailext-anonymous-editor' => 'A Fandom user',
+// 45 --------------------------------------------------------------------------
+'emailext-emailconfirmation-changed-footer-2' => 'See you on Fandom!',
+// 46 --------------------------------------------------------------------------
+'emailext-emailconfirmation-changed-subject' => 'Confirm your email address change on Fandom',
+// 47 --------------------------------------------------------------------------
+'emailext-emailconfirmation-changed-summary' => 'Please click the link below to confirm your change of email address on Fandom, so you can keep getting everything to fuel your fandom.',
+// 48 --------------------------------------------------------------------------
+'emailext-emailconfirmation-community-team' => 'The Fandom Community Team',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "The Fandom Community Team" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)